development-vm/replication/replicate-data-local.sh

```diff
@@ -7,37 +7,39 @@ set -eu
 
 . $(dirname $0)/common-args.sh
 
-if ! [ -x "$(command -v jq)" ]; then
-  echo 'Error: jq is not installed. https://stedolan.github.io/jq/' >&2
-  exit 1
-fi
-
-# setup AWS access
-SESSION_NAME=$(whoami)-$(date +%d-%m-%y_%H-%M)
-ROLE_ARN=$(awk '/role_arn/ {print $3}' ~/.aws/config)
-MFA_ARN=$(awk '/mfa_serial/ {print $3}' ~/.aws/config)
-
-if [[ "${MFA_TOKEN}" == "" ]]; then
-  echo "Error: must provide token"
-  echo "MFA_TOKEN=token $0"
-  exit 1
-fi
-
-unset AWS_SESSION_TOKEN
-
-CREDENTIALS=$(aws sts assume-role \
-                --role-session-name $SESSION_NAME \
-                --role-arn $ROLE_ARN \
-                --serial-number $MFA_ARN \
-                --token-code $MFA_TOKEN)
-
-if [[ $? == 0 ]]; then
-  echo "export AWS_ACCESS_KEY_ID=$(echo ${CREDENTIALS} | jq -r '.Credentials | .AccessKeyId')"
-  echo "export AWS_SECRET_ACCESS_KEY=$(echo ${CREDENTIALS} | jq -r '.Credentials | .SecretAccessKey')"
-  echo "export AWS_SESSION_TOKEN=$(echo ${CREDENTIALS} | jq -r '.Credentials | .SessionToken')"
-  status "Logging in with AWS_ACCESS_KEY_ID=$(echo ${CREDENTIALS} | jq -r '.Credentials | .AccessKeyId') AWS_SECRET_ACCESS_KEY=$(echo ${CREDENTIALS} | jq -r '.Credentials | .SecretAccessKey') AWS_SESSION_TOKEN=$(echo ${CREDENTIALS} | jq -r '.Credentials | .SessionToken')"
-else
-  exit "Unable to get AWS access"
+if ! ($SKIP_DOWNLOAD) then
+  if ! [ -x "$(command -v jq)" ]; then
+    echo 'Error: jq is not installed. https://stedolan.github.io/jq/' >&2
```

Comment:
We're a bit inconsistent about how we log errors. We've got `echo "foo" >&2`, a plain `echo "foo"` on line 22, and `error "foo"` [elsewhere](https://github.com/alphagov/govuk-puppet/blob/c1d3371f0233cd3631272b8dcc0a7d994e0e1bec/development-vm/replication/sync-aws-mysql.sh#L41). Do you know which one we should be using?